File: interviewbit/math/fizzbuzz.js
/*
    interviewbit
    track: "programming",
    section: "math",
    problem title: "FizzBuzz"
*/

module.exports = { 
 //param A : integer
 //return a array of strings
	fizzBuzz : function(A){
        var fbArr = [];
        for(var i=1;i<=A;i++){
            var str = "";
            if(i%3===0){
                str+="Fizz";
            }
            if(i%5===0){
                str+="Buzz";
            }
            if(str===""){
                str = i.toString();
            }
            // console.log(str);
            fbArr.push(str);
        }
        return fbArr;
	}
};

File: interviewbit/math/fizzbuzz.py

    # interviewbit
    # track: "programming",
    # section: "math",
    # problem title: "fizzBuzz"    

class Solution:
    # @param A : integer
    # @return a list of strings
    def fizzBuzz(self, A):
        answerList = [];
        for i in range(1,A+1):
            string = "";
            if(i%3==0):
                string+="Fizz";
            if(i%5==0):
                string+="Buzz";
            if(len(string)==0):
                string+=str(i);
            answerList.append(string);
        return answerList;